### DISCUSSION TITLE
Longhorn latest version compatibility with k8s

### DISCUSSION THREAD

**Discussion by adi088** [longhorn/longhorn#9481] [Q&A]
Hello Team,

As per official longhorn version 1.7.1 best practices, it is tested upto k8s version 1.28

I am planning to use Longhorn latest version 1.7.1 with k8s version 1.29. Is this going to cause compatibility issues or should we go with kubernetes 1.28 as mentioned in official document?

Please suggest.

**Comment by derekbit** [comment]
Hello @adi088 
We've tested 1.30.0 ([ref](https://ci.longhorn.io/job/public/job/v1.7.x/job/v1.7.x-longhorn-tests-sles-amd64/69/parameters/)). Thanks for the notification. We will update the document.
